charts/seed-bootstrap/templates/prometheus/networkpolicy.yaml

```diff
@@ -14,13 +14,7 @@ spec:
       app: prometheus
       role: monitoring
   egress:
-  - to:
-    - podSelector:
-        matchLabels:
-          component: node-exporter
-      namespaceSelector:
-        matchLabels:
-          kubernetes.io/metadata.name: kube-system
+  - to: []
```

Comment:
Should we add a note here or in the description at the top why a `podSelector` cannot be specified?

Reply:
Thank you! I added a comment, PTAL.